ln1_f4.py

```diff
@@ -8,74 +8,171 @@
 import subprocess
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+from email.mime.base import MIMEBase
+from email import Encoders
+from openpyxl import Workbook
+from openpyxl.styles import colors, Font, Border, Side
 
+database_file_name = "/home/lax/PycharmProjects/learn1/database4"
+add_database_file_name = "/home/lax/PycharmProjects/learn1/add_database4"
+excel_table_name = "/home/lax/PycharmProjects/learn1/new method test.xlsx"
 
-def get_sysresinfo(command):  # функція виконує консольну команду та повертає результат
+
+def get_sysresinfo(command):  # Executes console command and returns data as list
     command_data = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
     (command_data, err) = command_data.communicate()
-    command_data = command_data.replace(",", ".")  # Заміна ком на крапки для коректного виконання float()
-    return command_data.split()
+    return command_data.replace(",", ".").split()
 
 
-raw = get_sysresinfo("mpstat")  # обробка даних по завантаженості процесора
-cpu_user = float(raw[21])
-cpu_sys = 0
-for x in raw[22:29]:
-    cpu_sys += float(x)
-cpu_total = cpu_user + cpu_sys
-cpu_idle = float(raw[30])
+def get_cpu_info():  # return list format - cpu_user, cpu_sys, cpu_total, cpu_idle
+    cpu_usage_info = []
+    cpu_console_data = get_sysresinfo("mpstat")
+    cpu_usage_info.append(float(cpu_console_data[21]))
+    cpu_sys = 0
+    for x in cpu_console_data[22:29]:
+        cpu_sys += float(x)
```

Comment:
I can't use SUM function insted of simple loop because list contains "string" elements not "int" or "float" so i neet to turn them in float first.